### PR TITLE
Milestone 2. Mobile sections

### PR DESCRIPTION
**Pull request by indigodavid**
Modified the following details:
 - Added sections 'featured project', 'projects', and 'about me' to index.html
 - Styled sections following the Figma template
 - Added placeholders for the 'projects section'

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,15 +18,130 @@
...
             <li><a href="#"><img src="./images/github.svg" alt="GitHub"></a></li>
-            <li><a href="#"><img src="./images/linked-in.svg" alt="Linked In"></a></li>
+            <li><a href="#"><img src="./images/linked-in.svg" alt="LinkedIn"></a></li>
             <li><a href="#"><img src="./images/peace.svg" alt="Peace"></a></li>
             <li><a href="#"><img src="./images/twitter.svg" alt="Twitter"></a></li>
             <li><a href="#"><img src="./images/m.svg" alt="M"></a></li>
         </ul>
     </section>
+    <main>
+      <h1>My Recent Works</h1>
```
Comment (OLIPLICHE) [inline index.html]: ## Optional
`<h1>` defines the most important heading. and it is used only once on the page. You can change it with h2

- [ ]  Please open this link to learn more about h1 to h6 tag: [h1 to h6 tags](https://www.w3schools.com/tags/tag_hn.asp)

File: index.html [hunk 2]
```diff
@@ -18,15 +18,130 @@
...
+            <li>Ruby</li>
+          </ul>
+          <a href="#" class="button">See Project</a>
+        </div>
+      </section>
+      <section id="about-me">
+        <div class="about-head">
+          <h2>About me</h2>
+          <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <a href="#" class="button fs">Get My Resume</a>
```
Comment (OLIPLICHE) [inline index.html:112]: - [ ] Please the button "Get my resume" follows the button styling: **Active**, **Hover**, and **Pressed** 
- [ ] see design: [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

-
ho

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @indigodavid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Pull request, well defined 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (indigodavid) [reply index.html:112]: Hello, I corrected the project. Check it out, please. Best regards. 🙂

**Review by lily-coder — APPROVED** [review]
Hi @indigodavid , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**